app/src/main/java/com/keylesspalace/tusky/components/domainblocks/DomainBlocksFragment.kt

```diff
@@ -0,0 +1,108 @@
+package com.keylesspalace.tusky.components.domainblocks
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.lifecycleScope
+import androidx.paging.LoadState
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.google.android.material.snackbar.Snackbar
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.databinding.FragmentDomainBlocksBinding
+import com.keylesspalace.tusky.di.Injectable
+import com.keylesspalace.tusky.di.ViewModelFactory
+import com.keylesspalace.tusky.util.hide
+import com.keylesspalace.tusky.util.show
+import com.keylesspalace.tusky.util.viewBinding
+import com.keylesspalace.tusky.util.visible
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class DomainBlocksFragment : Fragment(R.layout.fragment_domain_blocks), Injectable {
```

Comment:
Should the user be able to refresh this, with a swipe, or a menu item?

Reply:
yes, I'd add this in a separate PR